feature/feature.go

```diff
@@ -39,6 +39,7 @@ const (
 	//
 	// alpha: v0.3
 	// beta: v0.4
+	// GA: v1.10
```

Comment:
This linter finding is 💩  Let's ignore it using `.golangci.yml`

Reply:
Oh, this regex ` - '^ \+.*'` didn't pick it up

Reply:
Seriously considering just disabling that linter